net-wireless/sdrpp/sdrpp-1.2.0_p20240826.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR=emake
+inherit cmake
+
+DESCRIPTION="Runtime dependencies for SDRPlusPlus"
+HOMEPAGE="
+	https://www.sdrpp.org/
+	https://github.com/AlexandreRouma/SDRPlusPlus/
+	"
+
+SRC_URI="https://github.com/jharvell/SDRPlusPlus/archive/refs/tags/v${PV}.tar.gz -> ${P}.tgz"
+
+LICENSE="GPL-3"
+SLOT=0
+KEYWORDS="~amd64 ~x86"
+IUSE="+airspy atv +audio-sink +audio-src badgesdr +bladerf +hackrf +hermes +limesdr +m17 pager +plutosdr portaudio-sink +portaudio-new-sink +rtlsdr ryfi +sched +sdrplay soapysdr uhd"
+PATCHES=(
+	"${FILESDIR}/${PN}-multilib-strict.patch"
```

Comment:
Please add a version of the package to the patch name. Also ```PATCHES``` is wrongly placed. It should be placed after ```*DEPEND```.